src/core/Target.ml

```diff
@@ -12,18 +12,52 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
  * LICENSE for more details.
  *)
-(*****************************************************************************)
-(* Prelude *)
-(*****************************************************************************)
-(* A target file.
- *
- * See AST_generic.program for more information. This module just
- * defines an alias to have better naming convetion to differentiate
- * a target program from a pattern program.
- *)
 
-(*****************************************************************************)
-(* Types *)
-(*****************************************************************************)
+(* See Target.mli for documentation of public items. *)
+
+type target_path = { origin : Origin.t; internal_path_to_content : Fpath.t }
```

Comment:
since the file is already called Target.ml, you can rename target_path to just path, so qualified it looks like Target.path.

Reply:
Also when the origin will not be a File, what will be internal_path_to_content? A path to a local tmp file?
Then would be good to add such a comment, explaining the difference
between the File path.t in origin and internal_path_to_content.

Reply:
I believe this is addressed in the documentation in the mli: https://github.com/semgrep/semgrep/blob/176e9e6b6aa263d12c9dc52180293c9505ba25c9/src/core/Target.mli#L76-L95

are you looking for something beyond what is said there?